test/mock/mock-lazyloaded-file.js

```diff
@@ -0,0 +1 @@
+window.lazyLoadingWorking = true;
```

Comment:
I can give a try to requirejs.undef to see it that triggers the module loading again when requested.